Libraries/LibJS/Runtime/ObjectConstructor.cpp

```diff
@@ -108,47 +108,23 @@ Value ObjectConstructor::set_prototype_of(Interpreter& interpreter)
 
 Value ObjectConstructor::get_own_property_descriptor(Interpreter& interpreter)
 {
-    if (interpreter.argument_count() < 2)
```

Comment:
What happens if we call `getOwnPropertyDescriptor()` with zero or one argument?

Reply:
- Zero: First argument (`target`) is `undefined` and the function throws `Uncaught exception: [TypeError]: First argument of Reflect.getOwnPropertyDescriptor() must be an object`
- One: Depends. Second argument (the property name) is `undefined` and will be stringified - so `"undefined"`
  So:
  - `Reflect.getOwnPropertyDescriptor({})` - `undefined`
  - `Reflect.getOwnPropertyDescriptor({ undefined: 123 })` - `{ "value": 123, "writable": true, "enumerable": true, "configurable": true }`
  - `Reflect.getOwnPropertyDescriptor("foo")` - `Uncaught exception: [TypeError]: First argument of Reflect.getOwnPropertyDescriptor() must be an object`

That's valid and even tested in the tests :)